### PR TITLE
Hi ma'am, i have added more screenshots of app, please merge it

### PR DESCRIPTION
**Pull request by amitYadavDev**

### CODE REVIEW

**Comment by saumya1singh** [comment]
@amit6663  great work

**Comment by saumya1singh** [comment]
just few more changes and you will earn your first reward .

File: README.md
Comment (saumya1singh) [inline README.md:12]: change all to some @amit6663

File: README.md
Comment (saumya1singh) [inline README.md:17]: @amit6663  change the image sizes . it's looking quite big . make it look small and concised .

**Review by saumya1singh — CHANGES_REQUESTED** [review]

**Comment by saumya1singh** [comment]
@amit6663  well done , your work is very appreciating 🎉

**Comment by saumya1singh** [comment]
@amit6663  I will approve your changes just after a quick review .

**Comment by saumya1singh** [comment]
@amit6663  I want you to remove some of the screens from screenshots that are not well designed.
I am writing the picture number which you have to remove : 7,11,12,13,14,16,17,18,19,23,24,28 ,26,27 .

When you guys will work on improving the screens then we will add more SS . 

And last thing is  squash your commits into one commit .

Then i will approve your changes and you will earn your point for KOSS contest .

**Comment by amitYadavDev** [comment]
I have done ma'am please look again, @SaumyaSingh1

**Comment by saumya1singh** [comment]
You will be rewarded once I approve @amit6663

**Review by saumya1singh — APPROVED** [review]

**Comment by saumya1singh** [comment]
@amit6663 approved 🎉

**Pull request closed without merging**
